nextgen/api_pipelines.go

```diff
@@ -562,33 +796,34 @@ func (a *PipelinesApiService) ListPipelines(ctx context.Context, org string, pro
 	return localVarReturnValue, localVarHttpResponse, nil
 }
 /*
-PipelinesApiService Update a Pipeline
-Updates a Pipeline.
+PipelinesApiService Move Pipeline YAML from inline to remote
+Creates a remote entity by fetching pipeline YAML from Harness.
  * @param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
- * @param body Pipeline request body
  * @param org Organization identifier
  * @param project Project identifier
  * @param pipeline Pipeline identifier
- * @param optional nil or *PipelinesApiUpdatePipelineOpts - Optional Parameters:
-     * @param "HarnessAccount" (optional.String) -  Slug field of the account the resource is scoped to. This is required for Authorization methods other than the x-api-key header. If you are using the x-api-key header, this can be skipped.
-@return PipelineCreateResponseBody
+ * @param optional nil or *PipelinesApiMoveConfigOpts - Optional Parameters:
+     * @param "Body" (optional.Interface of PipelineMoveConfigRequestBody) - 
+     * @param "HarnessAccount" (optional.String) -  Identifier field of the account the resource is scoped to. This is required for Authorization methods other than the x-api-key header. If you are using the x-api-key header, this can be skipped.
+@return PipelineMoveConfigResponseBody
 */
 
-type PipelinesApiUpdatePipelineOpts struct {
+type PipelinesApiMoveConfigOpts struct {
+    Body optional.Interface
     HarnessAccount optional.String
 }
 
-func (a *PipelinesApiService) UpdatePipeline(ctx context.Context, body PipelineUpdateRequestBody, org string, project string, pipeline string, localVarOptionals *PipelinesApiUpdatePipelineOpts) (PipelineCreateResponseBody, *http.Response, error) {
```

Comment:
make sure nothing is deleted from this file.Only add method that you need i guess that would just be ImportPipelineFromGit